app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  after_save :posts_count
+
```

Comment:
Well done adding the `after_save` Rails Active Callback to call the `#update_post_counter` method every time a new post has been saved to the database. 👏🏾👏🏾👏🏾